shared/AppInsightsCommon/src/Offline.ts

```diff
@@ -20,7 +49,13 @@ export function createOfflineListener(parentEvtNamespace?: string | string[]): I
     let _document = getDocument();
     var _navigator = getNavigator();        // Gets the window.navigator or workerNavigator depending on the global
     let _isListening: boolean = false;
-    let _onlineStatus: boolean = true;
+    let listenerList: OfflineCallback[] = [];
+
+    // Set the initial state
+    let _onlineStatus: boolean = _isNavOnline();
+    let rState: eOfflineValue = eOfflineValue.Online;
```

Comment:
So we don't need (and should not be using) this variable.

Reply:
I seperate the rstate and onlinestatus, so that they could be send back in the callback without any need of type changing.